### PR TITLE
[IMP] Link new picking to original sale order; [ADD] tests

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

File: pos_picking_load/tests/test_pos_picking_load.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                'product_uom_qty': 2,
+            })],
+            'order_policy': 'picking',
+        })
+        sale_order.action_button_confirm()
+        self.assertTrue(sale_order.picking_ids)
+        sale_order.picking_ids.picking_type_id.write({
+            'available_in_pos': True})
+
+        config = self.env.ref('point_of_sale.pos_config_main').copy()
```
Comment (legalsylvain) [inline pos_picking_load/tests/test_pos_picking_load.py:25]: Why don't you use original demo config ?

File: pos_picking_load/models/pos_order.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ def create_from_ui(self, orders):
         res = super(PosOrder, self).create_from_ui(orders)
         orders_with_original_picking = self.search([
             ('id', 'in', res), ('origin_picking_id', '!=', False),
-            ('state', 'not in', [('draft')])])
+            ('state', '!=', 'draft')])
```
Comment (legalsylvain) [inline pos_picking_load/models/pos_order.py:43]: Thanks for the clean.
could you too remove line ° 33, I forgot to remove ?

**Review by legalsylvain — APPROVED** [review]
Hi @StefanRijnhart,
Thanks a lot for this improvment 
code review and functional test on runbot. LGTM. Thanks !

File: pos_picking_load/tests/test_pos_picking_load.py (hunk 1 above)
Comment (StefanRijnhart) [reply pos_picking_load/tests/test_pos_picking_load.py:25]: I don't want to run into problems with sessions that are already in use (when I am performing manual tests on the same database).

File: pos_picking_load/models/pos_order.py (hunk 2 above)
Comment (StefanRijnhart) [reply pos_picking_load/models/pos_order.py:43]: Done!

File: pos_picking_load/tests/test_pos_picking_load.py (hunk 1 above)
Comment (legalsylvain) [reply pos_picking_load/tests/test_pos_picking_load.py:25]: I understand, it's a mess to have only a single closed demo session. Nice workaround.

**Comment by hparfr** [comment]
👍

**Review by hparfr — APPROVED** [review]

**Comment by legalsylvain** [comment]
Merged. Thanks @StefanRijnhart.

**Pull request merged**